data/actions/scripts/other/largeseashell.lua

```diff
@@ -0,0 +1,24 @@
+function onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	if player:getStorageValue(43000) <= os.time() then
+		local chance = math.random(100)
+		local msg = ""
+		if chance <= 16 then
+			player:addHealth(-200)
+			msg = "Ouch! You squeezed your fingers."
+		elseif chance > 16 and chance <= 64 then
+			Game.createItem(math.random(7632,7633), 1, player:getPosition())
+			msg = "You found a beautiful pearl."
+		else
+			msg = "Nothing is inside."
+		end
+		player:say(msg, TALKTYPE_MONSTER_SAY, false, player, item:getPosition())
+		item:transform(7553)
+		player:setStorageValue(43000, os.time() + 72000)
+		item:getPosition():sendMagicEffect(CONST_ME_BUBBLES)
+		return true
+	else
```

Comment:
`else` is not needed since you are returning in the if-block.

Reply:
![2019-12-31 23_00_07-Window](https://user-images.githubusercontent.com/4684880/71637399-52a04700-2c21-11ea-9e74-355ff0d11865.png)
this then?